### DISCUSSION TITLE
How to implement custom routes using express server

### DISCUSSION THREAD

**Discussion by naseef0** [vercel/next.js#15336] [Help]
![Screenshot 2020-07-20 at 10 50 57 PM](https://user-images.githubusercontent.com/45784156/87966842-7c488f00-cadb-11ea-83dc-b77b4c544713.png)

How to  change the browser url to `"/MAYBELLINE-SSMI-X-Ashley-Longshore-10-Dreamer/WATSPROD0009.html"` when hitting `"/product/:slug"` using server.js

**Comment by jamesmosier** [comment]
Do you want to redirect the user to that new page entirely? If so, you could use `res.redirect(newUrl);`

**Reply by naseef0** [reply]
How to do without redirect? if we redirect to a new page it will rerender. i don't need that. I need when a hit coming to  `/product/:slug`  it should load `/product/:slug"` but browser url should like this `/MAYBELLINE-SSMI-X-Ashley-Longshore-10-Dreamer/WATSPROD0009.html`
